src/components/Navigation/components/Item/tests/Item.test.tsx

```diff
@@ -100,6 +100,65 @@ describe('<Nav.Item />', () => {
       const link = item.find(UnstyledLink);
       expect(link.exists()).toBe(true);
     });
+
+    it('renders a badge if prop is provided', () => {
+      const spy = jest.fn();
+      matchMedia.setMedia(() => ({addListener: spy}));
+      const item = mountWithAppProvider(
+        <Item
+          label="some label"
+          url="/admin/orders"
+          badge="1"
+          subNavigationItems={[
```

Comment:
Is this or the field below needed for the test?

Reply:
Nope. This is just for the subnav items. Bad copy-pasta.